gwpy/segments/segments.py

```diff
@@ -144,6 +144,25 @@ def coalesce(self):
         return self
     coalesce.__doc__ = segmentlist.coalesce.__doc__
 
+    def to_table(self):
+        """Convert this `SegmentList` to a `~gwpy.table.Table`
```

Comment:
this class is really just `astropy.table.Table`
```suggestion
        """Convert this `SegmentList` to a `~astropy.table.Table`
```